source/agora/consensus/ValidatorSet.d

```diff
@@ -788,7 +681,7 @@ unittest
 
     // make test blocks used for restoring validator set
     Block[] blocks;
-    ulong last_height = ValidatorSet.ValidatorCycle;
+    ulong last_height = Enrollment.ValidatorCycle;
```

Comment:
There's a lot of diff coming from simply moving this constant. I think it should be its own commit.

Reply:
I can do it. but in some commit, there are two ValidatorCycle constants because EnrollmentPool must reference it. In the end, the ValidatorCycle will only have existed in the `Enrollment` class.